controller/transfer/project_transfer_manager.py

```diff
@@ -82,19 +82,19 @@ def ensure_project_exists(project_id: str) -> None:
 
 
 def import_sample_project(
-    user_id: str, organization_id: str, project_name: str
+    user_id: str, organization_id: str, project_name: str, project_type: str
 ) -> Project:
-    if project_name == "Clickbait - initial":
+    if project_type == enums.SampleProjectType.CLICKBAIT_INITIAL.value:
         file_name = "sample_projects/clickbait_initial.zip"
-    elif project_name == "Clickbait":
+    elif project_type == enums.SampleProjectType.CLICKBAIT.value:
         file_name = "sample_projects/clickbait.zip"
-    elif project_name == "AG News - initial":
+    elif project_type == enums.SampleProjectType.AG_NEWS_INITIAL.value:
         file_name = "sample_projects/ag_news_initial.zip"
-    elif project_name == "AG News":
+    elif project_type == enums.SampleProjectType.AG_NEWS.value:
         file_name = "sample_projects/ag_news.zip"
-    elif project_name == "Conversational AI - initial":
+    elif project_type == enums.SampleProjectType.CONVERSATIONAL_AI_INITIAL.value:
         file_name = "sample_projects/conversational_ai_initial.zip"
-    elif project_name == "Conversational AI":
+    elif project_type == enums.SampleProjectType.CONVERSATIONAL_AI.value:
         file_name = "sample_projects/conversational_ai.zip"
     else:
         raise Exception("Unknown sample project")
```

Comment:
This is a valid solution. Another way to approach this would be to use a "switch case" construct, which would be more natural for me when evaluating multiple conditions of which only one can be true. Python had no out-of-the-box switch-case construct for a long time and it got first introduced in python 3.10 if I remember correctly, so it could be that it would not even work for us to make use of the construct because I am not sure if we have a python version > 3.10 everywhere, but still wanted to mention it here just in case. So maybe no real todo for this comment but still good to think about imo

Good example of python switch-case can be found here: https://www.freecodecamp.org/news/python-switch-statement-switch-case-example/

Reply:
Yes, the switch case would be more natural, I tried that but it says exactly as you say. 

<img width="490" alt="Screenshot 2023-07-19 at 11 19 20" src="https://github.com/code-kern-ai/refinery-gateway/assets/104008550/05cb422a-9151-4290-9959-9c5d2e52189e">